src/components/TreeMenu/TreeMenuItem.js

```diff
@@ -24,28 +25,32 @@ const TreeMenuItem = ({
 		props.onClick && props.onClick();
 	}
 
+
 	return (
-		<li
-			{...props}
-			active="false"
-			className={`tree-menu-item${selected}${disabled}`}
-			style={{ paddingLeft: `${paddingLeft}rem` }}
-			onClick={e => {handleClick(e)}}
-		>
-			{(type == "folder") && (
-				<div
-					style={{ display: 'inline-block' }}
-					// Clickable toggle icon, not restricted as handleClick method
-					onClick={e => {
-						e.stopPropagation();
-						hasNodes && toggleNode && toggleNode();
-					}}
-				>
-					<ToggleIcon on={isOpen} selected={selected} />
-				</div>
-			)}
-			{label}
-		</li>
+		isDisabled && (resources.indexOf(resource) == -1) && (resources.indexOf("authz:superuser") == -1) ?
```

Comment:
@aringocode Are you aware of that this feature will hide all the content in the Treemenu when e.g. session will expire?  I think this has to be handled a bit different way. Also `resource` should not be a required parameter (we are using that also in Configuration UI).

Reply:
@Pe5h4 I tested, if the session ends, the items don't disappear anywhere

Reply:
@aringocode Are you sure, even tho when the testing user does not have superuser rights or correct resource?

Reply:
Yeah, I'm sure. These screenshots show resources for a user without access to disabled items and superuser. As you can see in these screenshots, the items did not disappear after the session ended.

Without edit library resource:
![image](https://github.com/TeskaLabs/asab-webui/assets/65866093/2b9b750f-670d-449b-a1e4-d5d04aa81457)
![image](https://github.com/TeskaLabs/asab-webui/assets/65866093/b46e9ee4-3da2-4dea-8841-04d47b18f833)



Superuser:
![image](https://github.com/TeskaLabs/asab-webui/assets/65866093/f85c786a-b960-45ca-8766-012252535dde)

![image](https://github.com/TeskaLabs/asab-webui/assets/65866093/402d4377-2e58-45fc-ad33-b5b9305f38fe)

Reply:
Alright, good. Test that also in `Configuration` part of the UI please :)